src/commands/app/doctor.ts

```diff
@@ -216,17 +216,10 @@ export default class AppDoctor extends Kommand {
     }
 
     this.log("Docker checks");
-    try {
-      const docov = await execute("docker-compose", "-v");
-      const matches = docov.stdout.match(/[^0-9.]*([0-9.]*).*/);
-      if (matches === null || matches.length === 0) {
-        this.logKo("Docker Version cannot be found");
-      } else {
-        this.logOk(`Docker Compose Version: ${matches[1]}`);
-      }
-    } catch (error: any) {
-      this.logKo("Docker Compose cannot be found");
-      suggestions.push("Install Docker Compose with 'npm run install:docker'");
+    const result = await checkPrerequisites(this);
+
+    if (!result) {
+      suggestions.push("Install Docker and the Compose plugin");
     }
 
     this.log(`----------------- DoKtor finish his job ! -----------------`);
```

Comment:
One last thing :sweat_smile: 

```suggestion
    this.log(`----------------- DoKtor finished its job ! -----------------`);
```